src/Raven.Server/Documents/Sharding/ShardedContext.cs

```diff
@@ -114,5 +155,46 @@ public bool HasClientConfigurationChanged(long clientConfigurationEtag)
             var actual = Hashing.Combine(lastClientConfigurationIndex, _lastClientConfigurationIndex);
             return actual > clientConfigurationEtag;
         }
+
+        public void UpdateMapReduceIndexes(Dictionary<string, IndexDefinition> mapReduceIndexes)
```

Comment:
is this used outside of ctor? why it is public?

Reply:
changed